Dwolla.Client/Models/Responses/BasegetResponse.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System.Collections.Generic;
```

Comment:
Filename should have capital 'g' in "Get"

Reply:
Had to do this in two commits cause github wouldn't take it.